docs/developer-guide.md

```diff
@@ -1,8 +1,115 @@
 # Developer Guidelines
 
-## Setting up the commit-msg hook for the pre-commit
+## Setting up the Local Development Environment
 
-Run `pre-commit install --hook-type commit-msg` to install the commit-msg hook.
+### Prerequisites
+
+Ensure the following are installed:
+
+- [Postgres](https://www.postgresql.org/)
+- [Sqitch](https://sqitch.org/) (requires a working Perl 5 installation)
+- [Node](https://nodejs.org/)
+- [asdf](https://asdf-vm.com/#/core-manage-asdf) (using your system's package manager & follow instructions to add it to your shell)
+
+### Repository and Version Control
+
+- Ensure you have push access (most likely by being added to the **@bcgov/cas-developers** GitHub team) to [bcgov/cas-cif](https://github.com/bcgov/cas-cif).
+- Ensure you have [GPG commit signing](https://docs.github.com/en/github/authenticating-to-github/signing-commits) set up on your local environment.
+  - First, ensure your `git config user.email` is set to the email address you want to use for signing.
+  - You can verify it's working when you commit to a branch and the signature is indicated by `git log --show-signature`. Once pushed, a "Verified" badge appears next to your commits on GitHub.
+- Clone a local copy of [bcgov/cas-cif](https://github.com/bcgov/cas-cif).
+
+### Install the Dev Tools
+
+In the root of `cas-cif`, run `make install_dev_tools`. This creates the databases, installs packages, etc.
+
+To seed the dev data, run `make drop_db && make deploy_dev_data`. If you get warnings about the database being accessed, stop and start postgres (`pg_ctl stop` then `pg_ctl start`).
+
```

Comment:
If you run into an error `failed to start: "No such file or directory"` this can be caused by the client definition in the global sqitch config. Remove the line `client = path/to/psql` from `~/.sqitch/sqitch.conf` 


This was an issue I ran into during my setup. Could be useful to add